src/sql-editor/standardSql/statementPositionResolversRegistry.ts

```diff
@@ -106,6 +106,22 @@ export function initStatementPositionResolvers(): StatementPositionResolversRegi
       resolve: (currentToken, previousKeyword, previousNonWhiteSpace, previousIsSlash) =>
         Boolean(previousNonWhiteSpace?.value.toLowerCase() === FROM),
     },
+    {
+      id: StatementPosition.AfterSchema,
+      name: StatementPosition.AfterSchema,
+      resolve: (currentToken, previousKeyword, previousNonWhiteSpace, previousIsSlash) => {
+        // depending on weather the schema was the last token in the query or not, current token might be whitespace or dot. if whitespace, just use the previous token
+        if (currentToken?.isWhiteSpace() && currentToken?.next) {
+          currentToken = currentToken?.previous;
+          previousNonWhiteSpace = currentToken?.getPreviousNonWhiteSpaceToken();
+        }
+        return Boolean(
+          currentToken?.isIdentifier() &&
+            currentToken?.value.endsWith('.') &&
+            previousNonWhiteSpace?.value.toLowerCase() === FROM
+        );
+      },
+    },
```

Comment:
From the BigQuey pov, where you query like `SELECT * from 'project.dataset.id'`, this will cause the after schema suggestions to kick in.